Formula/redis@4.0.rb

```diff
@@ -0,0 +1,72 @@
+class RedisAT40 < Formula
+  desc "Persistent key-value database, with built-in net interface"
+  homepage "https://redis.io/"
+  url "http://download.redis.io/releases/redis-4.0.11.tar.gz"
+  sha256 "fc53e73ae7586bcdacb4b63875d1ff04f68c5474c1ddeda78f00e5ae2eed1bbb"
+
+  keg_only :versioned_formula
+
+  option "with-jemalloc", "Select jemalloc as memory allocator when building Redis"
```

Comment:
Please remove option. See #31510 for the rationale behind removing options from formulas.